recycler/src/main/java/com/elpassion/android/commons/recycler/RecyclerViewAdapterCompositor.kt

```diff
@@ -0,0 +1,33 @@
+package com.elpassion.android.commons.recycler
+
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import com.elpassion.android.commons.recycler.components.ItemsStrategy
+import com.elpassion.android.commons.recycler.components.base.ItemAdapter
+
+class RecyclerViewAdapterCompositor<T : ItemAdapter<out RecyclerView.ViewHolder>>(private val itemsStrategy: ItemsStrategy<T>,
+                                                                                  private val getItemIdent: (position: Int) -> Long = { 0L },
+                                                                                  init: (RecyclerViewAdapterCompositor<T>.() -> Unit) = {}) : RecyclerView.Adapter<RecyclerView.ViewHolder>() {
+
+    init {
+        init()
+    }
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): RecyclerView.ViewHolder {
+        val itemView = LayoutInflater.from(parent.context).inflate(viewType, parent, false)
```

Comment:
It is a pity we can't use extension from from **view** module here... 😞 Methods count matters in android, but this is probably one reason to merge these modules or add a dependency in the future.